### PR TITLE
Style Event event and event cards to figma and style for mobile

### PR DESCRIPTION
**Pull request by Gordoburrito**
# Description

Mobile styling.
Also fixed some things that were in the design but not in the build. 

Add container components. Container mobile styling in the about mission purpose PR

# Links

Related pull requests:

Design:

# Additional Info

### CODE REVIEW

File: src/components/Cards/CardEvent/CardEvent.scss [hunk 1]
```diff
@@ -15,6 +16,7 @@
     justify-content: center;
     width: 4.5rem;
     height: 4.5rem;
+    border-radius: 0px 0px 4px 0px;
```
Comment (mattlaguardia) [inline src/components/Cards/CardEvent/CardEvent.scss:19]: this seems weird why one corner?

**Review by mattlaguardia — APPROVED** [review]
just address the one comment

File: src/components/Cards/CardEvent/CardEvent.scss (hunk 1 above)
Comment (Gordoburrito) [reply src/components/Cards/CardEvent/CardEvent.scss:19]: image
all 4 corners

Comment (Gordoburrito) [reply src/components/Cards/CardEvent/CardEvent.scss:19]: image

**Pull request merged**